app_dart/lib/src/service/config.dart

```diff
@@ -184,7 +184,33 @@ class Config {
   String get releaseCandidateBranchPath =>
       'bin/internal/release-candidate-branch.version';
 
-  DatastoreDB get db => _db;
+  /// Whether to read and write to Datastore.
+  ///
+  /// If `false`, the [db] will throw `UnsupportedError`, and as such usages
+  /// of Datastore should be guarded with if-statements and early terminations:
+  /// ```dart
+  /// // OK
+  /// if (config.useLegacyDatastore) {
+  ///   final db = config.db;
+  /// }
+  ///
+  /// // OK
+  /// if (!config.useLegacyDatastore) {
+  ///   return;
+  /// }
+  /// final db = config.db;
+  /// ```
+  bool get useLegacyDatastore => true;
```

Comment:
Could change to _getSingleValue('COCOON_USE_LEGACY_DATASTORE') - we'd use it just for a short while, but it would give us the "flip the bit" faster than rolling?

Reply:
Sounds good. Done!